index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile Vesrion</title>
+</head>
+<body>
+<div class="container">
+    <div class="toolbar">
+        <p id="welcome">Welcome</p>
+        <div class="menu">
+            <img src="images/menu.svg" alt="menu_icon">
+        </div>
+        </div>
+        <div class="headline_section">
+        <img src="images/header-illsutration-mobile.svg" alt="background_image">
+        <div class="text">
+        <div class="primary-text">
+            <span id="first">Hey there. I’m Mariane</span><br><span id="second">I’m a software developer</span>
+        </div>
+        <div class="second-text">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        </div>
+        </div>
+        <div class="link">
+            <img src="images/twitter.svg" alt="twitter_icon">
+            <img src="images/Linkedin icon.svg" alt="Linkedin_icon">
+            <img src="images/medium.svg" alt="medium_icon">
+            <img src="images/github.svg" alt="github_icon">
+            <img src="images/angellist.svg" alt="angellist_icon">
+        </div>
+    </div>
```

Comment:
- Kindly for the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).:grinning: :grinning: :grinning: 

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

check the example below :+1: 
```
<ul>
            <li> <a href=""><img src="images/twitter.png" alt="Twitter logo" /></a></li>
            <li><a href=""><img src="images/twitter.png" alt="Twitter logo" /></a></li>
            <li><a href=""><img src="images/git.png" alt="Git logo" /></a></li>
            <li><a href=""><img src="images/m.png" alt="M logo" /></a></li>
            <li><a href=""><img src="images/nice.png" alt="V logo" /></a></li>
</ul>
```

Reply:
Changes are made as suggested. Please check.